src/layouts/GlossaryTerms/termsList.module.css

```diff
@@ -1,6 +1,6 @@
 .list {
   margin-top: 3rem;
-  display: grid;
+  display: block;
   grid-template-columns: 1fr 30fr 1fr;
```

Comment:
Just a note, these aren't needed anymore, but I think they will be needed again once we fix the style of the Term component.